lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -417,8 +417,8 @@ sub run_backup {
         die $request->{_locale}->text('Error creating backup file');
 
     if ($request->{backup_type} eq 'email') {
-        # suppress warning of single usage of $LedgerSMB::Sysconfig::...
-        no warnings 'once';
```

Comment:
You are wrong; there is no such warning.  That is why I am against the dynamic code in Sysconfig.pm.

I would give up Perl for the lack of this warning.

Witness that `Used::ok` is used okay:

use strict;
use warnings;

package Used;
use vars '$ok';

$ok = 1;
$Used::never = 2;

package main;

my $v = $Used::ok;
$v = $Used::misspelled;

Name "Used::misspelled" used only once: possible typo at ./once line 13.
Name "Used::never" used only once: possible typo at ./once line 8.